lib/fs/ntfs/attrib_index_root.rb

```diff
@@ -91,15 +91,12 @@ def allocations=(indexAllocations)
       if @indexNodeHeader.hasChildren? && indexAllocations
         indexAllocations.each { |alloc| @indexAllocRuns << [alloc.header, alloc.data_run] }
       end
-      @indexAllocRuns
```

Comment:
We never check the return value of this method?

Reply:
Nor here.